src/android/SplashScreen.java

```diff
@@ -313,6 +315,22 @@ public void run() {
                     splashDialog.getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
```

Comment:
`splashDialog.getWindow()` is repeatedly used, so i think it could be extracted into a variable to avoid multiple method calls.